pkg/dss/auth/auth.go

```diff
@@ -229,9 +237,31 @@ func (a *Authorizer) AuthInterceptor(ctx context.Context, req interface{}, info
 		return nil, dsserr.PermissionDenied(fmt.Sprintf("missing scopes: %v", err))
 	}
 
+	if claims.Issuer == "" {
+		return nil, dsserr.Unauthenticated("missing issuer URI")
+	}
+
+	if err := a.checkExpired(claims.ExpiresAt); err != nil {
+		return nil, dsserr.Unauthenticated(fmt.Sprintf("%v", err))
+	}
+
 	return handler(ContextWithOwner(ctx, models.Owner(claims.Subject)), req)
 }
 
+func (a *Authorizer) checkExpired(claimedExpireTime int64) error {
+	now := time.Now()
+	if claimedExpireTime < now.Unix() {
+		return &tokenExpireError{msg: "Token Expired or Expiration missing"}
+	}
+
+	if claimedExpireTime > now.Add(time.Hour).Unix() {
+		return &tokenExpireError{
+			msg: "Token expiration time is too far in the furture, Max token duration is 1 Hour",
```

Comment:
@BenjaminPelletier can you comment on this piece with respect to the standards?

Reply:
One hour in the future is from the YAML text for AuthFromAuthorizationAuthority describing required fields in the JWT: "exp, with a time no further than 1 hour in the future."  So, this looks like a good test to me.